asn1acnconstants.h

```diff
@@ -30,6 +30,7 @@ namespace Constants {
 
 const char LANG_ASN1[] = "ASN.1";
 
+const char M_CONTEXT[] = "Asn1Acn.ContextMenu";
```

Comment:
`CONTEXT_MENU` or something :)

I just realized - it's all in a namespace, we can remove the "ASN" prefix from all the other constants here (I'll do it later)